setup.py

```diff
@@ -79,6 +108,7 @@ def get_extensions():
         "matplotlib",
         "tqdm>4.29.0",
         "tensorboard",
+        "fvcore @ git+https://github.com/facebookresearch/fvcore.git",
```

Comment:
remove this since this syntax does not always work (and should be addressed by #304)